### DISCUSSION TITLE
introduce minWidth:0 by default as part of css reset

### DISCUSSION THREAD

**Discussion by capaj** [chakra-ui/chakra-ui#8270] [Ideas]
### Description

as per title

This was originally proposed by tailwind author here:
https://twitter.com/adamwathan/status/1734696245015494711
and nicely explained here:
https://youtu.be/cH8VbLM1958?si=bjzFH5o5LFklJJ7U&t=60

### Problem Statement/Justification

![image](https://github.com/chakra-ui/chakra-ui/assets/1305378/430b35b5-0102-4358-9418-6076e18aee63)

prevent flebox overflows.
It's hard to always test in all screen sizes and with different browser default for min-width it's even impossible to test all scenarios on a single machine.
By applying this normalization we can get the same min-width for elements across all browsers overriding their hardcoded min-width values.

nice writeup here: https://www.joshwcomeau.com/css/interactive-guide-to-flexbox/#the-minimum-size-gotcha-11

### Proposed Solution or API

adding to CSS reset
```
* { minWidth: 0 }
```

### Alternatives

applying it in each chakra ui project manually myself

**Comment by segunadebayo** [comment]
We are not putting `min-width: 0` on every element in the reset.

Add it in your `globalCss` if you want that flex behavior.
